dynosaur/tests/pass/basic-rpitit.rs

```diff
@@ -0,0 +1,8 @@
+use dynosaur::dynosaur;
+
+#[dynosaur(DynMyTrait)]
+trait MyTrait {
+    fn foo(&self) -> impl Send;
```

Comment:
Try a method with two input lifetimes.. I don't expect the current desugaring to work with that.